app/controllers/concerns/saml_idp_logout_concern.rb

```diff
@@ -172,7 +171,7 @@ def prepare_saml_logout_request
 
   def finish_slo_at_idp
     sign_out_with_flash
-    redirect_to after_sign_out_path_for(:user)
+    redirect_to root_url
```

Comment:
The impact is that we don't have to go through this logic in Devise when we know that we will always redirect to the home page:
```ruby
def after_sign_out_path_for(resource_or_scope)
  scope = Devise::Mapping.find_scope!(resource_or_scope)
  router_name = Devise.mappings[scope].router_name
  context = router_name ? send(router_name) : self
  context.respond_to?(:root_path) ? context.root_path : "/"
end
```